frontend/src/metabase/components/ErrorPages/ErrorDiagnosticModal.tsx

```diff
@@ -266,12 +242,7 @@ export const ErrorExplanationModal = ({
       </Text>
       <Text my="md">
         {c("indicates an email address to which to send diagnostic information")
```

Comment:
i think this context is no longer relevant, also, we probably only need `t` instead of `jt` since we're no longer interpolating a link